Dockerfile

```diff
@@ -5,11 +5,11 @@ FROM gcr.io/distroless/static-debian12
 
 LABEL maintainer="The Perses Authors <perses-team@googlegroups.com>"
 
-USER nobody
+USER 65532:65532
```

Comment:
should we do the same for the Perses images ?

Reply:
Yes I'll create one PR for that. So far there are no issues when creating the Perses CR. But this might be an issue in more constrained environments